app/src/main/java/com/github/featuredetectandroid/utils/PhotoAnalyzer.kt

```diff
@@ -46,7 +47,11 @@ class PhotoAnalyzer(private val outputViewModel: OutputViewModel) : ImageAnalysi
             outputViewModel.featureDetector?.let { detector ->
                 if (detector.width != width) detector.width = width
                 if (detector.height != height) detector.height = height
+                val startTime = SystemClock.elapsedRealtime()
                 val (keypoints, _) = detector.detect(luminanceArrayToRGB(oriented))
+                if (outputViewModel.keypointOffsets.isNotEmpty()) {
```

Comment:
We want to show time if `outputViewModel.featureDetector != null`, so this check is redundant

Reply:
fixed